### ISSUE TITLE
Fix SimpleMenuPreference for support lib v28.0.0-alpha3

### ISSUE THREAD

**Issue by Gericop** [Gericop/Android-Support-Preference-V7-Fix#169]
The `SimpleMenuPreference` uses some extra elements to simulate the reserved icon space, however, from alpha3 the default value of reserving icon space is set to `true` but the preference ignores it.

![screenshot_1530702764](https://user-images.githubusercontent.com/5181621/42274225-2e9dac82-7f8c-11e8-913e-ea7f2dfe98c1.png)

**Comment by Mygod** [comment]
I mentioned the fix in my comment in another issue.

**Comment by Gericop** [comment]
Fixed in meanwhile by applying `Preference.Material` as the parent of `Preference.SimpleMenuPreference`.

**Comment by Mygod** [comment]
https://github.com/Gericop/Android-Support-Preference-V7-Fix/issues/111#issuecomment-396095797

**Comment by Mygod** [comment]
More precisely just put this in styles.xml: https://github.com/shadowsocks/shadowsocks-android/blob/8aabc73b9698b2bb26a89717d59ac1660e79fad8/mobile/src/main/res/values/styles.xml#L18-L25

**Comment by Gericop** [comment]
I think it's better to use the `Preference.Material` as the parent style since if the devs change it, they will be automatically propagated instead of manually copying the changed attributes.

**Comment by Mygod** [comment]
According to decompiled source in alpha 1 this is how other preferences do it, but maybe they have changed it.

**Comment by Gericop** [comment]
Yes, they do it like that, but if they change an attribute in alpha4 (or whatever the next release will be), you have to copy them over instead of just using the changed attributes directly.

**Comment by mc-12** [comment]
Hey guys, sorry that I have to ask another question on this topic. I am currently using version 28.0.0.0-alpha3. Setting `isIconSpaceReserved = false`  works just fine for settings, but not for categories. 
Example: Relating to the upper picture, "RingtonePreference" is aligned left but "Dynamic category" is aligned as `isIconSpaceReserved` was set to `true` (with extra spacing on the left). Do you have an idea what I'm doing wrong?


Btw: Awesome library!

**Comment by Gericop** [comment]
See #132 for details. This is a bug in the official lib, and this lib only has the fix in the AndroidX version (for the time being at least).

**Comment by mc-12** [comment]
Thanks a lot, I'll try it out.
